### ISSUE TITLE
How to get 3.6.0 instead of 3.6.3 into a pyenv

### ISSUE THREAD

**Issue by aktivkohle** [pyenv/pyenv#1027]
A library I am using does not like 3.6.3 it wants 3.6.0. I have tried everything.. `miniconda3-latest` contains 3.6.3. 

When I wrote that question, thought that putting a different miniconda3 into pyenv may have helped, but have since realised that the version numbers of miniconda do not correlate with the contained Python versions, for example `miniconda3-3.3.0` is not necessarily Python 3.3.0 cause there is a `miniconda3-4.3.30` but no Python 4. 

So ran `pyenv install 3.6.0` and in the project directory `pyenv local 3.6.0` but when I open the Python interpreter from that directory with the command `python` I see :

```
Python 3.6.3 |Anaconda, Inc.| (default, Oct 13 2017, 12:02:49) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```
Does anyone know how to get Python 3.6.0 into a pyenv, preferably as a miniconda?

**Comment by jimmywan** [comment]
https://conda.io/docs/user-guide/tasks/manage-python.html

**Comment by blueyed** [comment]
`pyenv shell 3.6.0` works, right?
You might not have the automatic handler for `pyenv local`'s activation installed.

**Comment by aktivkohle** [comment]
@jimmywan how are all those Conda commands relevant if you are using `miniconda3-latest` **inside** a pyenv ? As far as I understand there are several flavours of virtual environment for Python, and we are here as we have opted for this one 'pyenv'. I don't think that commands like `conda search` , `conda create` are relevant, but am open to learning something new.. 

@blueyed `pyenv shell 3.6.0` _is_ working now. I did however delete the miniconda folder and remove the miniconda relevant lines from .bashrc . That was temporarily installed as I was trying all kinds of things to change the version in that `miniconda3-latest`. 

The AWS ubuntu machine seems to have no system python. Tried to import the library after `pyenv shell 3.6.0` but didn't work.

Is it possible to get the `miniconda3-latest` in the pyenv to use 3.6.0 or is that actually impossible? If it's impossible will stop trying and try other means to fix this..
